manifests/system_table.pp

```diff
@@ -1,66 +1,62 @@
 # Add a system table $name to /etc/incron.d
 #
+# If multiple ``path`` and/or ``command`` options are specified, they will be
+# expanded into all matching possibilities.
+#
+# @example Multiplexed Path and Command
+#   incron::system_table { 'test':
+#     path    => ['/foo/bar', '/foo2/bar2'],
+#     command => ['/bin/baz', '/bin/baz2']
+#   }
+#
+#   Results in /etc/incron.d/test with contents:
+#     /foo/bar IN_MODIFY,IN_MOVE,IN_CREATE,IN_DELETE /bin/baz
+#     /foo/bar IN_MODIFY,IN_MOVE,IN_CREATE,IN_DELETE /bin/baz2
+#     /foo2/bar2 IN_MODIFY,IN_MOVE,IN_CREATE,IN_DELETE /bin/baz
+#     /foo2/bar2 IN_MODIFY,IN_MOVE,IN_CREATE,IN_DELETE /bin/baz2
+#
 # @option name
 #   The name of the table in /etc/incron.d/
+#
 # @param path
-#   Filesystem path to monitor
+#   Filesystem path(s) to monitor
+#
+#   * May contain Ruby ``Dir.glob`` compatible Strings
```

Comment:
Is that different than a shell glob?

Reply:
Technically yes, but shell globs will do what you expect.

Reply:
** is different in Ruby, so a good idea to point it out